package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-item-runner-qti",
-    "version": "0.22.0",
+    "version": "0.22.1",
```

Comment:
I thought the version was generated when publishing the package

Reply:
I will remove it when update from develop, thanks